include/seastar/core/reactor.hh

```diff
@@ -371,21 +380,17 @@ private:
         void handle_signal_once(int signo, noncopyable_function<void ()>&& handler);
         static void action(int signo, siginfo_t* siginfo, void* ignore);
         static void failed_to_handle(int signo);
-    private:
-        struct signal_handler {
-            signal_handler(int signo, noncopyable_function<void ()>&& handler);
-            noncopyable_function<void ()> _handler;
-        };
-        std::atomic<uint64_t> _pending_signals;
-        std::unordered_map<int, signal_handler> _signal_handlers;
```

Comment:
Why is this block moved upwards?

Reply:
To remove redundant `private` in a class

Reply:
Ill extract it into a different PR